pkg/kube/client.go

```diff
@@ -152,7 +152,7 @@ func (c *Client) Get(namespace string, reader io.Reader) (string, error) {
 		return "", err
 	}
 	err = perform(c, namespace, infos, func(info *resource.Info) error {
-		log.Printf("Doing get for: '%s'", info.Name)
+		log.Printf("Doing get for %s: '%s'", info.Mapping.GroupVersionKind.Kind, info.Name)
```

Comment:
I didn't accuse _you_ of adding it. I was just suggesting what I think would be an improvement to the same line of code that you're editing here (and below at line 393). It doesn't seem worthy of a separate PR, unless you know of many more places that would require the same quoting change.